### PR TITLE
Machine learning

### PR DESCRIPTION
**Pull request by alesgenova**

### CODE REVIEW

File: girder/molecules/server/calculation.py [hunk 1]
```diff
@@ -303,6 +302,8 @@ def _file_to_cjson(self, file, code='nwchem'):
                 cjson = json.loads(cjson)
             elif code == 'psi4':
                 cjson = cclib.convert_str(calc_data)
+            elif code == 'chemml':
+                cjson = json.loads(calc_data)
```
Comment (cjh1) [inline girder/molecules/server/calculation.py]: We might want to move this to a map pretty soon, keyed on the code.

File: girder/molecules/server/calculation.py [hunk 2]
```diff
@@ -313,6 +314,8 @@ def _extract_calculation_properties(self, cjson, calculation_data, code='nwchem'
             return self._extract_calculation_properties_nwchem(cjson, calculation_data)
         elif code =='psi4':
             return self._extract_calculation_properties_psi4(cjson)
+        elif code == 'chemml':
+            return self._extract_calculation_properties_chemml(cjson)
```
Comment (cjh1) [inline girder/molecules/server/calculation.py]: Likewise

File: girder/molecules/server/calculation.py [hunk 3]
```diff
@@ -267,8 +267,7 @@ def create_calc(self, params):
             file_id = file['_id']
             cjson, file_str = self._file_to_cjson(file, code)
 
-            if 'vibrations' in cjson or 'basisSet' in cjson:
-                props = self._extract_calculation_properties(cjson, file_str, code)
```
Comment (cjh1) [inline girder/molecules/server/calculation.py]: Why has this guard been removed?

Comment (alesgenova) [reply girder/molecules/server/calculation.py]: It wasn't 100% clear to me why it was there in the first place. The reason why I removed it now, is that when we ingest a ML calculation `vibrations` and `basisSet` will not be in the parsed cjson

**Pull request merged**